README.md

```diff
@@ -1,133 +1,31 @@
-# Julee Example - Docker Setup Guide
+# Julee
 
-This guide explains how to set up and run the Julee Example application using Docker Compose.
+Clean architecture for accountable and transparent digital supply chains.
 
-## Prerequisites
+Julee is a Python framework for building resilient, auditable business processes using Temporal workflows. Solutions are organized around your business domain—your bounded contexts become "accelerators" that speak your business language, not framework jargon.
 
-- Docker and Docker Compose installed
-- API keys for external services
+**Use Julee when:** processes must be done correctly, may be complex or long-running, need compliance audit trails (responsible AI, algorithmic due-diligence), or depend on unreliable services that may fail, timeout, or be rate-limited.
 
-## Quick Start
+**Core concepts:** Accelerators are collections of pipelines that automate a business area. Pipelines are use cases wrapped with Temporal, providing durability (survives crashes), reliability (automatic retries), observability (complete execution history), and supply chain provenance (audit trails that become "digital product passports").
 
-1. **Set up environment variables**:
-   ```bash
-   cp .env.example .env
-   ```
-   Edit `.env` and add your API keys:
-   ```bash
-   ANTHROPIC_API_KEY=your_anthropic_api_key_here
-   GEMINI_API_KEY=your_gemini_api_key_here
-   ```
```

Comment:
Later on you've got references to docker compose up to try it out, but that demo won't work without this data. If the demo is no longer relevant as we focus more on other things, we can remove it, but if we're not doing that now, we may need to keep instructions for running it here? Or somewhere?

Reply:
oh, you are right - will push a fix